src/services/apiMonitorInitializer.service.ts

```diff
@@ -0,0 +1,32 @@
+import { MongooseClient } from '../clients/mongoClient';
+import { UserAccountService } from '.';
+import { logger } from '../common/services/index';
+
+import { requestLogQueue } from '.';
```

Comment:
combine line 3 and 5 and use import {..} from "./index"

Reply:
logger is imported from common/services
that's a diff directory